android/src/main/java/com/AirMaps/AirMapView.java

```diff
@@ -100,6 +110,20 @@ public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float d
         });
 
         eventDispatcher = context.getNativeModule(UIManagerModule.class).getEventDispatcher();
+
+        this.mapLoadingTextView = new TextView(context);
+        this.mapLoadingTextView.setText("Loading map...");
```

Comment:
Is this standard?  What if people are building an app for non-english speakers?  Is there a platform loading indicator that we can use instead?